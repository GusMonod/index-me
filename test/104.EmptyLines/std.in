this file has

empty lines

but no new line at the end
