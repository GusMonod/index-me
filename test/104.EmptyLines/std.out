this: 0
file: 0
has: 0
empty: 2
lines: 2
but: 4
no: 4
new: 4
line: 4
at: 4
the: 4
end: 4
